staking/role/validator.go

```diff
@@ -0,0 +1,38 @@
+package role
+
+import (
+	"math/big"
+
+	"github.com/harmony-one/bls/ffi/go/bls"
+	"github.com/harmony-one/harmony/internal/common"
+)
+
+// Define validator staking related const
+const (
+	MaxNameLength            = 70
+	MaxIdentityLength        = 3000
+	MaxWebsiteLength         = 140
+	MaxSecurityContactLength = 140
+	MaxDetailsLength         = 280
+)
+
+// Validator - data fields for a validator
+type Validator struct {
+	Address           common.Address `json:"address" yaml:"address"`                         // ECDSA address of the validator
+	ValidatingPubKey  bls.PublicKey  `json:"validating_pub_key" yaml:"validating_pub_key"`   // The BLS public key of the validator for consensus
+	Description       Ties           `json:"description" yaml:"description"`                 // description for the validator
+	Active            bool           `json:"active" yaml:"active"`                           // Is the validator active in the validating process or not
+	Stake             *big.Int       `json:"stake" yaml:"stake"`                             // The stake put by the validator itself
+	UnbondingHeight   *big.Int       `json:"unbonding_height" yaml:"unbonding_height"`       // if unbonding, height at which this validator has begun unbonding
+	Commission        Commission     `json:"commission" yaml:"commission"`                   // commission parameters
+	MinSelfDelegation *big.Int       `json:"min_self_delegation" yaml:"min_self_delegation"` // validator's self declared minimum self delegation
+}
+
+// Ties - some possible IRL connections
+type Ties struct {
```

Comment:
Why change to "Ties"?  What does it mean? We should use commonly used terms for understandability.

Reply:
Ties means association, `I am tied to the land`, `What ties do you have to us?`